pkg/pillar/cmd/nodeagent/nodeagent.go

```diff
@@ -545,6 +547,11 @@ func handleLastRebootReason(ctx *nodeagentContext) {
 			ctx.rebootReason)
 		agentlog.DiscardCommonRebootReason()
 	}
+	ctx.panicTrace, ctx.panicTime = agentlog.GetPanicTrace()
+	if ctx.panicTrace != "" {
+		log.Errorf("shutdown at %s:\n %s", ctx.panicTime.String(), ctx.panicTrace)
+		agentlog.DiscardPanicTrace()
```

Comment:
It would be nice if we didn't loose this when there is a reboot loop and only delete once we know it has been sent to the cloud. 
Could he have logmanager pick up and send the file content and delete it when done?

Reply:
Makes sense. Will look into this.

Reply:
uploaded a new diff with changes